README.md

```diff
@@ -1,50 +1,48 @@
+I graduated with a Bachelor's in Chemistry from McMaster University, but have always had a keen interest in technology. I worked as a junior Software Developer at Select Vantage shortly afterward, creating python scripts and trading tools to be used in equity markets. I am currently finishing up my Master's degree in Computer Science and have joined Jarvis alongside. Since then, I have been collaborating with my team members in an Agile/Scrum environment to create backend applications. With that, I have acquired and solidified my knowledge by working on multiple projects, specifically with technologies such as Linux/Bash, RDMS/PostgresSQL, Dockers, Maven, etc. I am passionate about solving real-world problems and want to move my career forward as a Software Developer.
 
 ## Skills
 
-**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL, NoSQL/MongoDB, Agile/Scrum, Git
+**Proficient:** Java/Maven, Python, Linux/Bash, RDBMS/SQL, NoSQL/MongoDB, Agile/Scrum, Git
 
 **Competent:** JavaScript, React, Nodejs, HTML/CSS, GitFlow, Docker
 
-**Familiar:** GCP, C/C++, Neo4j, Maven, AWS
+**Familiar:** GCP, C/C++, Neo4j, AWS, Springboot
 
 ## Jarvis Projects
 
 Project source code: [https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid)
 
 
-**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/linux_sql)]: Implented a clustering monitoring program which automates the collects hardware and cpu usage information from the linux servers using Bash scripts and Crontab. The data was stored in a postgresSQL database that was initialized using Dockers. It was later than queried to perform analytics on overall usage and identify downtimes during high server usage. We utilzed Git as a version control tool and GitFlow branching strategy.
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/linux_sql)]: Implemented a clustering monitoring program that automates the collects hardware and CPU usage information from the Linux servers using Bash scripts and Crontab. The data was stored in a PostgreSQL database that was initialized using Dockers. It was later than queried to perform analytics on overall usage and identify downtimes during high server usage. We utilized Git as a version control tool and GitFlow branching strategy.
 
 **Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/core_java)]:
```

Comment:
Separate these core Java apps. The 'core java' unifying heading is only useful for training purposes: it looks better to have them as individual projects.